### DISCUSSION TITLE
可否增加deepseek v2 API的支持

### DISCUSSION THREAD

**Discussion by lannykov** [langgenius/dify#4136] [General]
如题。这个模型看起来不错，而且API的价格合适。

**Comment by gijigae** [comment]
DeepSeek V2 supports OpenAI compatible API. It means you can already use DeepSeek V2 in Dify as long as you acquire an API key (see the image attached) from https://platform.deepseek.com/ . Hope it helps.

<img width="1952" alt="image" src="https://github.com/langgenius/dify/assets/81459/0fd3a340-295c-4092-b58c-3cb31b66d529">
